include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -572,6 +573,15 @@ struct __global_scan_functor
             __out_acc[__item_idx] =
                 static_cast<typename __internal::__get_tuple_type<__in_type, __out_type>::__type>(__bin_op_result);
         }
+        __internal::__invoke_if_not(_Inclusive{}, [&]() {
+            //store an initial value to the output first position should be done as postprocess (for in-place scanning)
```

Comment:
I personally don't like these post-processing steps on the host. I know we do it for the reduction too, and it's not really related to this patch. But if the user specifies a device policy, we might want to think about executing all operations on the device. What we're doing right now can have unwanted side effects.

Reply:
Agree in General. But here there is not post-processing on the host.  "Post-processing" here means applying inti on the final scan phase. (Our scan SYCL backend pattern consist of 3 phase - local scan, within work group, scan between groups and final (global) scan.)